### PR TITLE
fix(dgraph): Add flags for blockcache and bloomcache size in alpha

### PR DESCRIPTION
**Pull request by ashish-goswami**
---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6184)

 

Docs Preview: [Dgraph Preview](https://dgraph-02fd6b5ece-85886.surge.sh)

### CODE REVIEW

**Comment by danielmai** [comment]
___
*[dgraph/cmd/alpha/run.go, line 116 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/6184#-MEdFjOU6sEGpGMd02rl:-MEdFjOU6sEGpGMd02rm:bdmtt9o) ([raw file](https://github.com/dgraph-io/dgraph/blob/afba28112fd141d4317526f14afa57a897c32263/dgraph/cmd/alpha/run.go#L116)):*
<details><summary><i alt="summary">Quoted 4 lines of code…</i></summary>

> ```
> 	flag.Int("badger.blockcache_size", 1024*1024*1024,
> 		"Size of block cache for posting dir badger.")
> 	flag.Int("badger.bloomcache_size", 512*1024*1024,
> 		"Size of bloom filter cache for posting dir badger.")
> ```
</details>

Let's set these in terms of MB, not bytes.

File: worker/server_state.go [hunk 1]
```diff
@@ -157,10 +157,10 @@ func (s *ServerState) initStorage() {
 		opt := badger.DefaultOptions(Config.PostingDir).
 			WithValueThreshold(1 << 10 /* 1KB */).
 			WithNumVersionsToKeep(math.MaxInt32).
-			WithMaxCacheSize(1 << 30).
+			WithMaxCacheSize(int64(Config.BlockCacheMB * 1024 * 1024)).
```
Comment (jarifibrahim) [inline worker/server_state.go:160]:
```suggestion
			WithMaxCacheSize(int64(Config.BlockCacheMB)).
```

File: worker/server_state.go [hunk 2]
```diff
@@ -157,10 +157,10 @@ func (s *ServerState) initStorage() {
...
 			WithKeepBlockIndicesInCache(true).
 			WithKeepBlocksInCache(true).
-			WithMaxBfCacheSize(500 << 20) // 500 MB of bloom filter cache.
+			WithMaxBfCacheSize(int64(Config.BloomCacheMB * 1024 * 1024))
```
Comment (jarifibrahim) [inline worker/server_state.go:163]:
```suggestion
			WithMaxBfCacheSize(int64(Config.BloomCacheMB))
```

File: worker/server_state.go (hunk 1 above)
Comment (jarifibrahim) [reply worker/server_state.go:160]: My bad.

File: worker/server_state.go (hunk 2 above)
Comment (jarifibrahim) [reply worker/server_state.go:163]: My bad.

**Pull request closed without merging**